### PR TITLE
Adding basic disabling code

### PR DESCRIPTION
**Pull request by willdoran**
This pull request makes the following changes:
- This provides a means to allow disabling of validation on data import via the web UI

Test checklist:
- [ ]

- [ ] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

File: application/classes/Controller/Api/CSV/Import.php [hunk 1]
```diff
@@ -33,7 +33,9 @@ public function action_post_index_collection()
 		 * Step two of import.
 		 * Support all line endings without manually specifying it (primarily added because of OS9 line endings which do not work by default )
 		 */
-		ini_set('auto_detect_line_endings', 1);
+        ini_set('auto_detect_line_endings', 1);
+        ini_set('memory_limit', '-1');ini_set('memory_limit', '-1');
```
Comment (rjmackay) [inline application/classes/Controller/Api/CSV/Import.php]: Why do this twice?

File: application/config/features.php [hunk 2]
```diff
@@ -78,5 +78,10 @@
 	// see TODO <docs link>
 	'redis' => [
 		'enabled' => FALSE,
+    ],
+    
+    // CSV Speedup
+	'csv-speedup' => [
+		'enabled' => TRUE,
```
Comment (rjmackay) [inline application/config/features.php]: This should default to false right?

Comment (willdoran) [reply application/config/features.php]: Yep

File: application/classes/Controller/Api/CSV/Import.php (hunk 1 above)
Comment (willdoran) [reply application/classes/Controller/Api/CSV/Import.php]: typo

File: src/Core/Usecase/ImportUsecase.php [hunk 3]
```diff
@@ -97,43 +112,43 @@ public function isSearch()
...
+        $this->verifyImportAuth($entity);
+
+        $new_status = 'PENDING';
+        $this->csv->setState([
+            'status' => $new_status
+        ]);
+
+        service('repository.csv')->update($this->csv);
+
+        $this->emit(
```
[the remark this answers is not in the document]
Comment (willdoran) [reply src/Core/Usecase/ImportUsecase.php]: That's weird it does seem to be async. The code will continue executing after the emit and return at the end of this method. Trying to find the league docs on events.

**Comment by rjmackay** [comment]
@willdoran @rowasc should this be abandoned now? Or is it still useful?

**Pull request closed without merging**